components/badge.css

```diff
@@ -54,10 +48,6 @@
   height: 1.2em;
   text-align: center;
   color: var(--secondary-theme-color);
-
-  position: absolute;
```

Comment:
repeated in same class

Reply:
I will definitely go through this and made changes accordingly

Reply:
Actually, that code part is deleted that's why it is shown in red background-color